examples/Cookbook.ipynb

```diff
@@ -479,6 +480,126 @@
+    "transfer_factory = TransferTransactionsFactory(config=TransactionsFactoryConfig(chain_id=\"D\"))\n",
+    "\n",
+    "alice = Address.new_from_bech32(\"erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th\")\n",
+    "bob = Address.new_from_bech32(\"erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx\")\n",
+    "\n",
+    "# With \"data\" field\n",
+    "transaction = transfer_factory.create_transaction_for_transfer(\n",
+    "    sender=alice,\n",
+    "    receiver=bob,\n",
+    "    native_amount=1000000000000000000,\n",
+    "    data=\"for the book\".encode()\n",
+    ")\n",
+    "\n",
+    "print(\"Transaction:\", transaction_converter.transaction_to_dictionary(transaction))\n",
+    "print(\"Transaction data:\", transaction.data.decode())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And for ESDT transfers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Transaction: {'nonce': 0, 'value': '0', 'receiver': 'erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th', 'sender': 'erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th', 'senderUsername': '', 'receiverUsername': '', 'gasPrice': 1000000000, 'gasLimit': 1484000, 'data': 'TXVsdGlFU0RUTkZUVHJhbnNmZXJAODA0OWQ2MzllNWE2OTgwZDFjZDIzOTJhYmNjZTQxMDI5Y2RhNzRhMTU2MzUyM2EyMDJmMDk2NDFjYzI2MThmOEAwMkA1NDQ1NTM1NDJkMzMzODY2MzIzNDM5QDAxQDAxQDQyNDE1MjJkNjMzODMwNjQzMjM5QEA4YWM3MjMwNDg5ZTgwMDAw', 'chainID': 'D', 'version': 2, 'options': 0, 'guardian': '', 'signature': '', 'guardianSignature': '', 'relayer': '', 'innerTransactions': []}\n",
+      "Transaction data: MultiESDTNFTTransfer@8049d639e5a6980d1cd2392abcce41029cda74a1563523a202f09641cc2618f8@02@544553542d333866323439@01@01@4241522d633830643239@@8ac7230489e80000\n"
+     ]
+    }
+   ],
+   "source": [
+    "first_token = Token(identifier=\"TEST-38f249\", nonce=1)\n",
+    "first_transfer = TokenTransfer(token=first_token, amount=1)\n",
+    "\n",
+    "second_token = Token(identifier=\"BAR-c80d29\")\n",
+    "second_transfer = TokenTransfer(token=second_token, amount=10000000000000000000)\n",
+    "\n",
+    "transaction = transfer_factory.create_transaction_for_transfer(\n",
+    "    sender=alice,\n",
+    "    receiver=bob,\n",
+    "    token_transfers=[first_transfer, second_transfer]\n",
+    ")\n",
+    "\n",
+    "print(\"Transaction:\", transaction_converter.transaction_to_dictionary(transaction))\n",
+    "print(\"Transaction data:\", transaction.data.decode())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively, the protocol now supports sending both native and esdt tokens in the same transaction. If a `native_amount` is provided together with `token_transfers`, the native token will also be included in the `MultiESDTNFTTrasfer` built-in function call."
```

Comment:
"now supports" is somehow incorrect. It will support. If there is an already published page on docs, we can link to it. Otherwise, let's use the future tense.

Reply:
As far as I've seen, there is no docs page yet, but I've changed the text to be:
```
the protocol will support (in the very near future) sending ...
```